tools/scripts-core/scripts/test.js

```diff
@@ -1,15 +1,16 @@
-const spawn = require('cross-spawn');
-const { resolveBin } = require('../utils/path-resolver');
-const { getPreset } = require('../utils/preset');
-const { getUserConfigFile } = require('../utils/env');
+/* eslint-disable import/extensions */
+import spawn from 'cross-spawn';
+import { resolveBin } from '../utils/path-resolver.js';
```

Comment:
those 3 imports are often needed, 
what about having an index.Js exposing them for being imported as a oneliner for convenience?